apps/api/pages/api/bookings/_get.ts

```diff
@@ -189,7 +189,7 @@ async function handler(req: NextApiRequest) {
       const userEmails = users.map((u) => u.email);
       args.where = buildWhereClause(userId, attendeeEmails, userIds, userEmails);
     } else if (filterByAttendeeEmails) {
-      args.where = buildWhereClause(userId, attendeeEmails, [], []);
+      args.where = buildWhereClause(null, attendeeEmails, [], []);
```

Comment:
This was always passing the admin's user ID meaning the results returned only included bookings where the admin was included in the booking.